### ISSUE TITLE
Target not found when using the script.

### ISSUE THREAD

**Issue by mochaccin** [prasanthrangan/hyprdots#658]
Hi, i tried running the script on a clean endeavourOS install following the Ksk royal's [youtube video](https://www.youtube.com/watch?v=mb8h1-LB9K0), but when i run the install script, i get the target not found error, to be more specific, it says this:

"error: target not found: network-manager-applet blueman qt5-wayland qt6-wayland qt5-quickcontrols qt5-quickcontrols2 qt5-graphicaleffects dunst waybar slurp swappy cliphist polkit-kde-agent xdg-desktop-portal-hyprland  jq qt5-imageformats ffmpegthumbs kde-cli-tools brightnessctl pamixer kuantum qt5ct kitty neofetch dolphin vim ark zsh eza"

Also i found comments on the video saying they have the same problem even in other distros.

Love your dotfiles <3.

**Comment by ShaSha146** [comment]
Same problem I’m able to manually install the packages but the install script won’t install them. Might be some sort of issue with the repos or mirrors being used. Best to try again after a day or two to see if the issue on the repo side resolves itself.

**Comment by Dewinz** [comment]
Having the same issue, also tried manually install all the packages. The script detected them, but not their dependencies. Therefore it didn’t work.

**Comment by 404GH0ST** [comment]
I didn't what the problem cause is. But in my case, I fixed it by changing these two line of codes in `install_pkg.sh`:
```bash
 sudo pacman ${use_default} -S $pkg_arch
 ```
 to
 ```bash
 bash -c "sudo pacman ${use_default} -S $pkg_arch"
 ```
 and
 ```bash
  $aurhlpr ${use_default} -S $pkg_aur
 ```
 to
 ```bash
 bash -c "$aurhlpr ${use_default} -S $pkg_aur"
 ```

**Comment by Dewinz** [comment]
This fix also works for me. Thanks!

**Comment by ShaSha146** [comment]
Resolved the issue by following Ksk royal's youtube video for arch Linux my best guess is one of the following fixed it for me

- Not selecting a mirror region on archinstall menu
- Selecting NetworkManager from the archinstall menu instead of copying the network settings from the arch iso
- Granting the Hyprdot install.sh script the right permissions by running “chmod +x ./install.sh” before running install.sh

**Comment by prasanthrangan** [comment]
Sorry guys, my bad it was the `IFS` modification that caused it.
but its fixed in ec57443, so please use the latest version.

**Comment by mochaccin** [comment]
Run a clean install and the script works smoothly, only problem that i noticed is that when i press Super + C it logs me out to the welcome screen instead of opening vs code. I recommended the repo to a friend and he gets the same bug when opening vs code. I tried opening on the terminal and the same happens.
